packages/webapp/src/views/developer/upload-metric-file.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'multipart-form-layout.html' %}
```

Comment:
@AjinkyaJawalkar 
The layout `developer-form-layout.html` doesn't have multipart-form attributes. See line no 12 of `multipart-form-layout.html` for your reference.